library/L3_Application/test/periodic_scheduler_test.cpp

```diff
@@ -53,6 +52,10 @@ TEST_CASE("Testing PeriodicScheduler", "[periodic_scheduler]")
 {
   using namespace rtos;  // NOLINT
```

Comment:
NIT: Using namespace is no longer necessary since the whole test case is wrapped in the `rtos` namespace.